modules/thunder_media/thunder_media.install

```diff
@@ -531,3 +531,68 @@ function thunder_media_update_8101() {
 
   return $updateLogger->outputHtml();
 }
+
+/**
+ * Introduce image and video paragraph.
+ */
+function thunder_media_update_8102() {
+
+  /** @var \Drupal\thunder\ThunderUpdateLogger $updateLogger */
+  $updateLogger = Drupal::service('logger.thunder_update');
+  $updateLogger->cleanLogs();
+
+  \Drupal::service('module_installer')->install([
+    'config_update',
+  ]);
+
+  $configUpdater = \Drupal::service('config_update.config_update');
+
+  $imports = [
+    'paragraphs_type' => ['image', 'video'],
+    'field_storage_config' => ['paragraph.field_image', 'paragraph.field_video'],
+    'field_config' => [
+      'paragraph.image.field_image',
+      'paragraph.video.field_video',
+    ],
+    'view' => ['video_browser'],
+    'system.simple' => ['entity_browser.browser.video_browser'],
+    'entity_form_display' => [
+      'paragraph.video.default',
+      'paragraph.image.default',
+    ],
+    'entity_view_display' => [
+      'paragraph.video.default',
+      'paragraph.image.default',
+      'paragraph.image.preview',
+      'paragraph.video.preview',
+    ],
+  ];
+
+  $error = FALSE;
+  foreach ($imports as $type => $entities) {
+    foreach ($entities as $entity) {
+      try {
+        $configUpdater->import($type, $entity);
+      }
+      catch (\Exception $e) {
+        $error = TRUE;
+        $updateLogger->warning(t('Unable to import config: @type -> @entity', ['@type' => $type, '@entity' => $entity]));
+      }
+    }
+  }
+
+  if (!$error) {
+    /** @var \Drupal\thunder_updater\Updater $thunderUpdater */
+    $thunderUpdater = \Drupal::service('thunder_updater');
+    $thunderUpdater->checkListPoints(['v1_1__image_video_paragraph_install']);
+  }
+
+  // Output log result depending on channel used for update execution.
+  if (function_exists('drush_log') && PHP_SAPI === 'cli') {
+    $updateLogger->outputDrush();
+
+    return '';
+  }
+
+  return $updateLogger->outputHtml();
+}
```

Comment:
Add 'access video_browser entity browser pages' to editor and seo role in the update hook.